### PR TITLE
chore: adding page titles using project title. 

### PR DESCRIPTION
**Pull request by sriramveeraghanta**
### This PR adds the title to each page to improve tab identification:

- We aim to enhance the user experience by updating page titles to improve tab identification while working with multiple tabs in the browser. 
- This enhancement enables users to effortlessly discern the content of each tab based on the project name or issue title. 
- By dynamically adjusting page titles to reflect relevant information, such as project names or issue titles, users can quickly identify and navigate between tabs, streamlining their workflow and reducing cognitive load. 
- This improvement ensures a more intuitive and efficient browsing experience, enhancing usability and productivity for users interacting with our application.

**Format:**

**Dashboard**: \<Workspace Name\> - Dashboard
**Analytics**: \<Workspace Name\> - Analytics
**Project Issues**: \<Project Name\> - Issues
**Project Cycles**: \<Project Name\> - Cycles
**Project Modules:** \<Project Name\> - Modules
**Project Pages:** \<Project Name\> - Pages
**Project Views:** \<Project Name\> - Views
**Issue Detail:**  \<Project Identifier\> - \<Issue Sequence Id\> \<Issue Title\>
**Project Settings:** Project - \<Settings Name\>
**Profile Settings:**  Profile - \<Settings Name\>
**Workspace Settings:**  \<Workspace Name\> - \<Settings Name\>
**God Mode:** God Mode - \<Settings Name\>
**Create Workspace:** Create Workspace
**Sign Up:** Sign Up
**Sign In:** Sign In
**Reset Password:** Reset Password
**Forgot Password:** Forgot Password
**Onboarding:** Onboarding

image

This PR is attached to [WEB-427](https://app.plane.so/plane/projects/02c3e1d5-d7e2-401d-a773-45ecba45d745/issues/9ff498de-1130-4729-8061-191657a43c13)

### CODE REVIEW

File: web/pages/[workspaceSlug]/active-cycles.tsx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 import { WorkspaceActiveCyclesUpgrade } from "components/workspace";
 // layouts
 import { AppLayout } from "layouts/app-layout";
 // types
 import { NextPageWithLayout } from "lib/types";
+// hooks
+import { useWorkspace } from "hooks/store";
 
-const WorkspaceActiveCyclesPage: NextPageWithLayout = () => <WorkspaceActiveCyclesUpgrade />;
+const WorkspaceActiveCyclesPage: NextPageWithLayout = () => {
```
Comment (sriramveeraghanta) [inline web/pages/[workspaceSlug]/active-cycles.tsx:13]: Add observer to the component.

File: web/pages/[workspaceSlug]/index.tsx [hunk 2]
```diff
@@ -2,12 +2,26 @@ import { ReactElement } from "react";
...
+import { PageHead } from "components/core";
 import { WorkspaceDashboardView } from "components/page-views";
 import { WorkspaceDashboardHeader } from "components/headers/workspace-dashboard";
 // types
 import { NextPageWithLayout } from "lib/types";
+// hooks
+import { useWorkspace } from "hooks/store";
 
-const WorkspacePage: NextPageWithLayout = () => <WorkspaceDashboardView />;
+const WorkspacePage: NextPageWithLayout = () => {
```
Comment (sriramveeraghanta) [inline web/pages/[workspaceSlug]/index.tsx:13]: Add observer to the component.

File: web/pages/[workspaceSlug]/projects/[projectId]/archived-issues/[archivedIssueId].tsx [hunk 3]
```diff
@@ -45,6 +45,9 @@ const ArchivedIssueDetailsPage: NextPageWithLayout = () => {
   );
 
   const issue = getIssueById(archivedIssueId?.toString() || "") || undefined;
+  const project = (issue?.project_id && getProjectById(issue?.project_id)) || undefined;
+  const pageTitle = project && issue ? `${project?.identifier}-${issue?.sequence_id} ${issue?.name}` : undefined;
```
Comment (sriramveeraghanta) [inline web/pages/[workspaceSlug]/projects/[projectId]/archived-issues/[archivedIssueId].tsx:49]: Add observer to the component to reflect these changes.

File: web/pages/[workspaceSlug]/projects/[projectId]/inbox/[inboxId].tsx [hunk 4]
```diff
@@ -34,26 +34,32 @@ const ProjectInboxPage: NextPageWithLayout = observer(() => {
       }
     }
   );
+  // derived values
+  const pageTitle = currentProjectDetails?.name ? `${currentProjectDetails?.name} - Inbox` : undefined;
```
Comment (sriramveeraghanta) [inline web/pages/[workspaceSlug]/projects/[projectId]/inbox/[inboxId].tsx:38]: Add observer to the component.

File: web/pages/[workspaceSlug]/projects/[projectId]/views/[viewId].tsx [hunk 5]
```diff
@@ -20,7 +21,12 @@ const ProjectViewIssuesPage: NextPageWithLayout = () => {
   const router = useRouter();
   const { workspaceSlug, projectId, viewId } = router.query;
   // store hooks
-  const { fetchViewDetails } = useProjectView();
+  const { fetchViewDetails, getViewById } = useProjectView();
+  const { getProjectById } = useProject();
+  // derived values
+  const projectView = viewId ? getViewById(viewId.toString()) : undefined;
+  const project = projectId ? getProjectById(projectId.toString()) : undefined;
+  const pageTitle = project?.name && projectView?.name ? `${project?.name} - ${projectView?.name}` : undefined;
```
Comment (sriramveeraghanta) [inline web/pages/[workspaceSlug]/projects/[projectId]/views/[viewId].tsx:29]: Add observer to the component.

**Review by pablohashescobar — APPROVED** [review]

**Pull request merged**